apa102.py

```diff
@@ -1,5 +1,6 @@
 import spidev
 import colorsys
+import numpy as np
```

Comment:
It would be good if we could not need the NumPy dependency for a single function - it's a meaty one. `colorsys` feels more reasonable.